extra_model/_adjectives.py

```diff
@@ -10,13 +10,14 @@
 def cluster_adjectives(adjective_counts, vectorizer):  # noqa: C901
     """cluster adjectives based on a constant radius clustering algorithm
     technical implementation uses a scikitlearn BallTree
-    parameters:
-    adjective_counts (dict): dictionary with adjectives and their counts
-    vectorizer (Vectorizer): provide embeddings to evaluate adjective similarity
-    returns:
-    tuple ([string],[int],[[(string,int)]]): first list contains the representative words for the adjective clusters,
+    :param adjective_counts: dictionary with adjectives and their counts
+    :type adjective_counts: [(str,int)]
+    :param vectorizer:  provide embeddings to evaluate adjective similarity
+    :type vectorizer: :class:`_vectorizer.vectorizer`
```

Comment:
Chris re-renamed `_vectorizer.vectorizer` to `Vectorizer`, so once you rebase to current version of main you should get couple of errors.